docs/openbsd/setup.md

```diff
@@ -39,11 +39,15 @@ A `GENERIC` kernel must be compiled with
 option enabled:
 
 ```sh
-$ cd /sys
-$ echo 'pseudo-device kcov 1' >arch/amd64/conf/KCOV
-$ echo 'include "arch/amd64/conf/KCOV" >>arch/amd64/conf/GENERIC
-$ make -C arch/amd64/compile/GENERIC config
-$ make -C arch/amd64/compile/GENERIC
+$ cd /sys/arch/amd64
+$ cat <<EOF >conf/SYZKALLER
+include "arch/amd64/conf/GENERIC"
+pseudo-device kcov 1
+EOF
+$ cp -R compile/GENERIC compile/SYZKALLER
+$ make -C compile/SYZKALLER obj
+$ make -C compile/SYZKALLER config
+$ make -C compile/SYZKALLER
```

Comment:
This much is working fine. I'm getting trouble with the following VM creation sequence (below is running as root, but I tried as syzkaller with similar results):
```
ci-openbsd# dmesg | grep vm
vmm0 at mainbus0: VMX/EPT (using slow L1TF mitigation)
ci-openbsd# uname -a
OpenBSD ci-openbsd.syzkaller 6.4 GENERIC.MP#293 amd64
ci-openbsd# vmctl status                                                                                                                           
   ID   PID VCPUS  MAXMEM  CURMEM     TTY        OWNER NAME
    1     -     1    512M       -       -    syzkaller syzkaller
ci-openbsd# cat /etc/vm.conf                                                                                                                       
vm "syzkaller" {
  disable
  disk "/syzkaller/syzkaller.img"
  local interface
  owner syzkaller
  allow instance { boot, disk, memory }
}
ci-openbsd# vmctl create "/syzkaller/syzkaller.img" -s 4G
vmctl: imagefile created
ci-openbsd# vmctl start syzkaller-1 -c -t syzkaller -b /bsd.rd -d "/syzkaller/syzkaller.img"
vmctl: start vm command failed: Device busy
```